assets/src/edit-story/components/canvas/layout.js

```diff
@@ -24,6 +24,10 @@ import { forwardRef, createRef } from 'react';
 /**
  * Internal dependencies
  */
+/**
+ * WordPress dependencies
+ */
```

Comment:
Need to clean up the comments a little bit

Reply:
Done in https://github.com/google/web-stories-wp/pull/3643/commits/331240015ac564463578ff42d828ff2667432cda